indra/newview/lloutfitslist.cpp

```diff
@@ -885,18 +885,10 @@ void LLOutfitListBase::refreshList(const LLUUID& category_id)
         ++items_iter)
     {
         LLViewerInventoryCategory *cat = gInventory.getCategory(*items_iter);
-        if (!cat)
+        if (cat)
         {
-            LLInventoryObject* obj = gInventory.getObject(*items_iter);
```

Comment:
This makes no sense. If getCategory failed, object would be an LLViewerInventoryItem. If said item is a link to a category, casting it to LLViewerInventoryCategory is going to cause issues.